04week/welcome-to-our-site/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Welcome to our site!</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media (max-width:576px){
+        body {width:576px}
+        .flex-container{background-color:#33a8dd}
+        .flex-contents{background-color:#f7f5f6}
+        .flex-contents{color:black}
+        .flex-contents{flex-direction:column}
+        .flex-contents{height:1200px}
+      }
+    </style>
+      </head>
+
+  <body>
+    <header role= "banner">
+      <div class= "flex-container">
+      <div class= "title">
+        <h1>Welcome to our site</h1>
+      </div>
+    </div>
+    </header>
+
+    <div class= "flex-contents">
+        <article id="social" class="column">
```

Comment:
id's should only be used on unique items; these should all have the same `class` or use a unique `id` for each one.. i.e. `id="social1"` etc..